sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -31,6 +31,14 @@
 from .. import tekton_api_version
 
 
+# pyyaml representer for literal yaml string dumper
```

Comment:
this could use more information like above

Reply:
added more detail comment